model/src/main/scala/uk/gov/ons/addressIndex/model/server/response/AddressResponse.scala

```diff
@@ -132,10 +132,18 @@ object AddressResponseAddress {
     * @param other
     * @return
     */
-  def fromPafAddress(maxScore: Float)(other: PostcodeAddressFileAddress): AddressResponseAddress =
+  def fromPafAddress(maxScore: Float)(other: PostcodeAddressFileAddress): AddressResponseAddress = {
+
+    val poBoxNumber = if (other.poBoxNumber == "") "" else s"PO BOX ${other.poBoxNumber}"
```

Comment:
You could use my StringUtil here.
It converts a `isEmpty` String to a `None` and anything else to a `Some(str)` you could then map it and do the interpolation.
result would look like so:
```
val poBoxNumber =  other.poBoxNumber.toOption.map(str => s"PO BOX $str").getOrElse("")